src/audio/google_rtc_audio_processing.c

```diff
@@ -225,14 +330,36 @@ static struct comp_dev *google_rtc_audio_processing_create(
 	if (!cd->tuning_handler)
 		goto fail;
 
-	cd->state = GoogleRtcAudioProcessingCreate();
+	if (MEMORY_BUFFER_BYTES > 0) {
```

Comment:
always true as this is hard coded

Reply:
It's written as if it can chance at runtime but that is weird given it's a constant

Reply:
The code is designed to work also when setting either GOOGLE_RTC_AUDIO_PROCESSING_MEMORY_BUFFER_STEREO_SIZE_BYTES
GOOGLE_RTC_AUDIO_PROCESSING_MEMORY_BUFFER_MONO_SIZE_BYTES to zero.

It looks slightly different now though